frontend/src/components/simple-auto-filter/simple-auto-filter.tsx

```diff
@@ -70,7 +75,7 @@ const SimpleAutoFilter: FC = () => {
   const isButtonVisible = Boolean(
     Object.values(filters).some((filter) => filter.length >= 1) ||
       brandDetails[0].brandId != '' ||
-      brandDetails[0].modelId != '',
+      brandDetails[0].modelIds.length,
   );
```

Comment:
Not memoized

Reply:
This value needs to be recalculated in every form change, so I think result with/without memo will be the same.

Reply:
I chose wrong name value "uniqueOptions". It sholud be `notChosenOptions`